src/OpenSage.Game/Gui/Apt/ActionScript/Opcodes/Function.cs

```diff
@@ -326,6 +329,10 @@ public override void Execute(ActionContext context)
             var args = FunctionCommon.GetArgumentsFromStack(context);
 
             FunctionCommon.ExecuteFunction(funcName, args, obj, context);
+
+            var result = context.Stack.Pop();
```

Comment:
Do you need to check if the function returns a result? Or do all functions return a result?

Reply:
That is actually a very good question. In case there is no return an "Undefined" should be pushed on the stack, which i don't do yet. Let me add this to this PR later.